.github/workflows/ci.yaml

```diff
@@ -11,28 +11,17 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Setup PATH
-        shell: bash
-        run: |
-          mkdir -p "$HOME/.local/bin"
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
```

Comment:
.local/bin is on path already, the folder doesn't exists though but can be created simply later.